sim/midas/src/main/scala/midas/widgets/ClockBridge.scala

```diff
@@ -151,11 +154,12 @@ class ClockTokenVector(numClocks: Int) extends Bundle with HasChannels with Cloc
   *
   * Target and host time measurements provided by simif_t are facilitated with MMIO to this bridge
   *
-  * @param clockInfo Clock frequency information for each target clock
+  * @param key Clock frequency information for each target clock
```

Comment:
Since ScalaDoc was provided for this class could you add some for `ClockKey` and update this accordingly.